app/views/users/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+
+<html>
```

Comment:
As mentioned by @maelfosso  the `html` tags are not necessary. We already have them in `app/views/layouts/application.html.erb` 
Let's get rid of them please.

Reply:
done